kernel/debian.series

```diff
@@ -1,21 +1 @@
-0102-cpuidle-skip-synchronize_rcu-on-single-CPU-systems.patch
-0103-sysrq-skip-synchronize_rcu-if-there-is-no-old-op.patch
-0104-fbcon-enable-no-blink-by-default.patch
-0105-vmstats-wakeups.patch
-0107-cgroup.patch
-0108-smpboot-reuse-timer-calibration.patch
-0109-perf.patch
-0110-pci-probe-identify-known-devices.patch
-0111-init-no-wait-for-the-known-devices.patch
-0112-ksm-wakeups.patch
-0113-init-do_mounts-recreate-dev-root.patch
-0114-xattr-allow-setting-user.-attributes-on-symlinks-by-.patch
-0201-crypto-allow-testmgr-to-be-skipped.patch
-0202-silence-Power-down-msg.patch
-0203-fs-9p-fix-create-unlink-getattr-idiom.patch
-0204-rdrand.patch
-0205-reboot.patch
-0206-no-early-modprobe.patch
-0208-Show-restart-information-using-info-log.patch
-0209-HACK-9P-always-use-cached-inode-to-fill-in-v9fs_vfs_.patch
-0002-UBUNTU-SAUCE-no-up-disable-pie-when-gcc-has-it-enabl.patch
```

Comment:
@erick0z could  you confirm that  removing this patch does not  conflict with Ubuntu packaging?

Reply:
yes, we need to check this for 16.10 builds. This was required to workaround the pie flag defaults introduced for gcc on 16.10.

Reply:
@jcvenegas These changes does not conflict with Ubuntu versions.

Reply:
@jcvenegas @devimc However, there is a minor issue (dependency problem) I found when building for RHEL and SUSE-Based distros. I will send a PR for that.